python/ray/projects/schema.json

```diff
@@ -60,5 +60,8 @@
       }
     }
   },
-  "required": ["name", "cluster"]
-}
+  "required": [
+    "name",
+    "cluster"
+  ]
+}
```

Comment:
We should have trailing newlines (also in the other files, maybe you can configure your editor to add it)